alerts/cluster-monitoring-operator/NodeFilesystemSpaceFillingUp.md

```diff
@@ -2,71 +2,101 @@
 
 ## Meaning
 
-The `NodeFilesystemSpaceFillingUp` alert triggers when two conditions are met:  
+This alert indicates that a filesystem on an OpenShift worker node (RHCOS) is
```

Comment:
again the alerting rules applies to all sort of nodes: not RHCOS, not worker, ...